ui/src/components/lib/instruction-block/instruction-block.tsx

```diff
@@ -0,0 +1,26 @@
+import { Button, Flex, FormStatus } from '@vkontakte/vkui'
+
+import { OutputField } from '@/components/lib/output-field'
+import { ConditionalRender } from '@/components/lib/conditional-render'
+
+import styles from './instruction-block.module.css'
+
+type InstructionBlockProps = {
+  title: string
+  copyableText: string
+  onOkClick?: () => void
+  isClosable?: boolean
+}
+
+export const InstructionBlock = ({ title, copyableText, onOkClick, isClosable = true }: InstructionBlockProps) => (
```

Comment:
I don't like the naming of this. Maybe 'CopyableBlock'? What does 'instruction' mean in this context?